src/plugins/data/server/query/route_types.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { SerializableRecord } from '@kbn/utility-types';
+
+/*
+ * These types are used to define the shape of the response from the saved query API
+ */
+
+// @ts-expect-error
```

Comment:
I don't know why this is necessary - this type is essentially copied from `MatchAllFilterMeta` where it works.

Reply:
I'm mostly confused because I made sure my copied `Filter` is also a `type` and yet the compiler still complains.